ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/config/Cli.scala

```diff
@@ -187,14 +194,20 @@ object Cli {
           filters = filters,
           beginOffset = beginOffset,
           endOffset = endOffset,
-          objectives = Some(
-            WorkflowConfig.StreamConfig.TransactionObjectives(
-              maxDelaySeconds = maxDelaySeconds,
-              minConsumptionSpeed = minConsumptionSpeed,
-              minItemRate = minItemRate,
-              maxItemRate = maxItemRate,
-            )
-          ),
+          objectives = {
+            (maxDelaySeconds, minConsumptionSpeed, minItemRate, maxItemRate) match {
```

Comment:
This wants a helper function to stay DRY

Reply:
5722bfb86199f66d90fd5c6d9bcdcb839c543e43